Commandant/Errors.swift

```diff
@@ -39,7 +39,11 @@ internal func informativeUsageError<T>(keyValueExample: String, option: Option<T
 		description += "]"
 	}
 
-	description += "\n\t\(option.usage)"
+	description += option.usage.componentsSeparatedByString("\n")
```

Comment:
> *all* "line terminators."

Ooh, found a better solution, but just after you merged this…

[Something like this](https://github.com/acmeism/RosettaCodeData/blob/a25938f123d08740ce128efb60c66c483a799e54/Task/Read-a-file-line-by-line/Objective-C/read-a-file-line-by-line-2.m) would do what we want.